torch/_inductor/autotune_process.py

```diff
@@ -857,7 +857,11 @@ def make_run_fn(
 
     def cleanup_run_fn(self) -> None:
         if self.DLL is not None:
-            self.DLL.close()
+            """
+            Check close attr due to it crash on Windows.
+            """
+            if hasattr(self.DLL, "close"):
+                self.DLL.close()
```

Comment:
I guess we need to check how to close the DLL properly on Windows?

Reply:
Sure, @jgong5 I will do some research to switch `CppBenchmarkRequest `'s  `CDLL` to `DLLWrapper`, and add Windows impl.